<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table_brron</class>
 <widget class="QDialog" name="table_brron">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Бронь</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>600</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string> Назад</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_actualBroni">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Предстоящие брони</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_find">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>600</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Оформить бронь</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_change">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>600</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить статус</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id_bron">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fio">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_status">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>550</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>---</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Завершен</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Не Завершен</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Отменен</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_studia">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>510</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>---</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Студия:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Статус</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>510</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Бронь оборудования</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата:          c</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Поиск по дате/изменение</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>540</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>до</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reset">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>600</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сбросить фильтры</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
